.NET/Microsoft.Recognizers.Text.Number/Constants.cs

```diff
@@ -47,11 +47,15 @@ public static class Constants
         public const string POWER_SUFFIX = "Pow";
         public const string SPECIAL_SUFFIX = "Spe";
 
+        // RelativeTo
+        public const string RELATIVE_START = "start";

```

Comment:
Why only one relative constant? There can be three of them, right?

Reply:
"current" and "end" is in the RelativeReferenceRelativeToMap.
Ordinal and Ordinal.relative have different implementation in this case.

Reply:
I understand, but I think we should define the constants so it's clear what the potential values are.